data/remnant missions.txt

```diff
@@ -495,3 +560,759 @@ mission "Remnant: Technology Available"
+			label end
+			`	Well, I need to get back to work. That last battle put us behind schedule on our repairs. Have a good day!" With a final flourish that you think is the equivalent of an exclamation mark, Tali strides out of the outfitters, heading towards another landing bay.`
+
+event "Remnant Salvage Available"
+	outfitter "remnant"
+		"Generator (Furnace Class)"
+		"Generator (Candle Class)"
+		"Triple Plasma Core"
+		"Small Heat Shunt"
+		"Large Heat Shunt"
+		"Systems Core (Small)"
+		"Thruster (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Steering (Planetary Class)"
+		"Thruster (Comet Class)"
+		"Steering (Comet Class)"
+		"Korath Fire-Lance"
+	
+
+mission "Remnant: Expanded Horizons Quarg 1"
+	name "Visiting the Quarg"
+	description "Take Remnant researchers to see some Quarg."
+	minor
+	passengers 3
+	blocked "This mission requires bunks for three passengers."
+	source
+		government "Remnant"
+	destination "Wayfarer"
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "First Contact: Quarg: offered"
+		random < 40
+	on offer
+		conversation
+			`As you relax in the cafeteria on <planet> you are approached by a young Remnant. Judging by the hand-held sensors and tools hanging off her belt, you suppose she is a researcher or an engineer.`
+			`	"Hello Captain <last>. My name is Dawn, and I have been spending the past few days skimming the archive you retrieved. I wanted to ask you a few questions. May I?"`
+			choice
+				`"Sure, pull up a chair."`
+				`"I'm not interested."`
+					decline
+			`	She quickly slides into the chair opposite you and pulls out a datapad. "Firstly, our histories mention how humanity met a race called the 'Quarg' and had some ongoing dialogs with them, but not much beyond that. Now here in this history that you brought us, it says that humanity has started exchanging technology with them! Is this true?" She looks at you intently. You sign negatively, and she dejectedly continues. "Could you explain about them, then?" You reply that the Quarg have allowed humanity to live near them, and that the Quarg are usually willing to answer questions, including scientific ones. You continue, elaborating that while the Quarg don't seem to mind having an innovative shipyard studying them as best it can, they don't actually share their technology with anyone.`
+			`	After several hours of telling her about meeting the Quarg and visiting their world it finally dawns on you that it would be easy enough to take her to visit the Tarazed system. You offer to take her and a couple of her colleagues. She eagerly accepts, although she expresses a preference for landing on the human world, so they can observe the Quarg ships from a distance, as they would rather not draw any attention to themselves. She runs out of the cafeteria after telling you they will meet you at your ship in half an hour.`
```

Comment:
`it finally dawns`
Can we not, given her name is Dawn?
`You offer to take her and a couple of her colleagues.`
This should be a choice ("I'll tell you about the Quarg" doesn't just automatically lead into "Let me bring you to the Quarg"), perhaps with consequences. I'm not sure how fond the Remnant as a whole would be for this outsider to start taking younger Remnant citizens out of the Wastes, possibly putting them in danger.

Reply:
Changed to "it occurs to you."

Also changed that to be a choice.